include/seqan3/core/metafunction/range.hpp

```diff
@@ -220,18 +230,20 @@ using innermost_value_type_t = typename innermost_value_type<t>::type;
  * Attention, this metafunction implicitly removes cv-qualifiers and reference from the types it recurses on and
  * returns.
  */
+template <typename t>
+constexpr size_t dimension_v = 0;
+
```

Comment:
Why do you want to add this anchor that makes the statement valid for types where it makes no sense? Is this required for something?